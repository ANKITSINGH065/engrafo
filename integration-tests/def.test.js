var utils = require("./utils");

test("simple \\def without arguments render correctly", done => {
  utils.expectBodyToMatchSnapshot("def/no_arg.tex", done);
});

test("\\def multiple simple arguments render correctly", done => {
  utils.expectBodyToMatchSnapshot("def/many_simple_args.tex", done);
});

test("\\def with control seq suffix renders correctly", done => {
  utils.expectBodyToMatchSnapshot("def/control_seq_suffixed.tex", done);
});
