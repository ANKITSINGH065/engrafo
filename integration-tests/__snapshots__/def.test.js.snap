// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\\def multiple simple arguments render correctly 1`] = `
<body>
  <dt-article
    class="centered"
  >
    <p>
      one one two two three three four four five five six six seven seven after
    </p>
  </dt-article>
  <dt-appendix>
    <div
      class="l-body"
    />
  </dt-appendix>
  <div
    id="cite-hover-boxes-container"
  />
  <div
    id="footnote-hover-boxes-container"
  />
</body>
`;

exports[`\\def with control seq suffix renders correctly 1`] = `
<body>
  <dt-article
    class="centered"
  >
    <blockquote>
      <p>
        <span>
          hello
        </span>
        <code>
          world
        </code>
      </p>
    </blockquote>
  </dt-article>
  <dt-appendix>
    <div
      class="l-body"
    />
  </dt-appendix>
  <div
    id="cite-hover-boxes-container"
  />
  <div
    id="footnote-hover-boxes-container"
  />
</body>
`;

exports[`simple \\def without arguments render correctly 1`] = `
<body>
  <dt-article
    class="centered"
  >
    <p>
      bar
    </p>
  </dt-article>
  <dt-appendix>
    <div
      class="l-body"
    />
  </dt-appendix>
  <div
    id="cite-hover-boxes-container"
  />
  <div
    id="footnote-hover-boxes-container"
  />
</body>
`;
